src/deepsparse/open_pif_paf/annotate.py

```diff
@@ -0,0 +1,233 @@
+  --source TEXT                   File path to an image or directory of image
+                                  files, a .mp4 video, or an integer (i.e. 0)
+                                  for webcam  [required]
+  --engine [deepsparse|onnxruntime|torch]
+                                  Inference engine backend to run on. Choices
+                                  are 'deepsparse', 'onnxruntime', and
+                                  'torch'. Default is 'deepsparse'
+  --image_shape, --image_shape INTEGER
+                                  Image shape to use for inference, must be
+                                  two integers  [default: 384, 384]
+  --num_cores, --num-cores INTEGER
+                                  The number of physical cores to run the
+                                  annotations with, defaults to using all
+                                  physical cores available on the system. For
+                                  DeepSparse benchmarks, this value is the
+                                  number of cores per socket
+  --save_dir, --save-dir DIRECTORY
+                                  The path to the directory for saving results
+                                  [default: annotation-results]
+  --name TEXT                     Name of directory in save-dir to write
+                                  results to. defaults to
+                                  {engine}-annotations-{run_number}
+  --target_fps, --target-fps FLOAT
+                                  Target FPS when writing video files. Frames
+                                  will be dropped to closely match target FPS.
+                                  --source must be a video file and if target-
+                                  fps is greater than the source video fps
+                                  then it will be ignored
+  --no_save, --no-save            Set flag when source is from webcam to not
+                                  save results.Not supported for non-webcam
+                                  sources  [default: False]
+  --help                          Show this message and exit
+
+#######
+Examples:
+
+1) deepsparse.open_pif_paf.annotate --source PATH/TO/IMAGE.jpg
+2) deepsparse.open_pif_paf.annotate --source PATH/TO/VIDEO.mp4
+3) deepsparse.open_pif_paf.annotate --source 0
+4) deepsparse.open_pif_paf.annotate --source PATH/TO/IMAGE_DIR
+"""
+import logging
+from typing import Optional, Tuple
+
+import click
+
+import cv2
+from deepsparse.open_pif_paf.utils import annotate_image
+from deepsparse.pipeline import Pipeline
+from deepsparse.utils.annotate import (
+    annotate,
+    get_annotations_save_dir,
+    get_image_loader_and_saver,
+)
+from deepsparse.utils.cli_helpers import create_dir_callback
+
+
+open_pif_paf_default_stub = None
+
+DEEPSPARSE_ENGINE = "deepsparse"
+ORT_ENGINE = "onnxruntime"
+TORCH_ENGINE = "torch"
+
+_LOGGER = logging.getLogger(__name__)
+
+
+@click.command()
```

Comment:
Will allow you to remove the underscore & hyphens options, AND the show_default=True
```suggestion
@click.command(
    context_settings=dict(
        token_normalize_func=lambda x: x.replace("-", "_"), show_default=True
    ),
)
```